src/main/java/io/vertx/ext/mail/MailConfig.java

```diff
@@ -603,6 +614,88 @@ public MailConfig setUserAgent(String userAgent) {
     return this;
   }
 
+  /**
+   * Is DKIM enabled, defaults to false.
+   *
+   * @return enableDKIM
+   */
+  public boolean isEnableDKIM() {
+    return enableDKIM;
+  }
+
+  /**
+   * Sets true to enable DKIM Signatures, sets false to disable it.
+   *
+   * <p>
+   *     This is used most for temporary disable DKIM without removing DKIM opations from current config.
+   * </p>
+   *
+   * @param enableDKIM if DKIM Singature should be enabled
+   * @return this to be able to use the object fluently
+   */
+  public MailConfig setEnableDKIM(boolean enableDKIM) {
+    this.enableDKIM = enableDKIM;
+    return this;
+  }
+
+  /**
+   * Gets the DKIM options.
+   *
+   * @return dkimSignOptions
+   */
+  public List<DKIMSignOptions> getDkimSignOptions() {
+    return dkimSignOptions;
+  }
+
+  /**
+   * Adds a DKIMSignOptions.
+   *
+   * @param dkimSignOptions the DKIMSignOptions
+   * @return this to be able to use the object fluently
+   */
+  public MailConfig addDKIMOption(DKIMSignOptions dkimSignOptions) {
```

Comment:
instead `addDKIMSignOptions`

Reply:
Done by renaming to `addDKIMSignOption` because adding methods are not allowed to end with `s`